mock_test.go

```diff
@@ -111,25 +112,25 @@ func (n *mockNSQD) handle(conn net.Conn) {
 	for idx < len(n.script) {
 		select {
 		case line := <-readChan:
-			log.Printf("mock: %s", line)
+			n.t.Logf("mock: %s", line)
 			n.got = append(n.got, line)
 			params := bytes.Split(line, []byte(" "))
 			switch {
 			case bytes.Equal(params[0], []byte("IDENTIFY")):
 				l := make([]byte, 4)
 				_, err := io.ReadFull(rdr, l)
 				if err != nil {
-					log.Printf(err.Error())
+					n.t.Logf(err.Error())
```

Comment:
this should use `Log()` instead of `Logf()` (and 2 other cases below)

Reply:
OK, fine